website/docs/assets/templates/capd-template.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: capi.weave.works/v1alpha1
+apiVersion: capi.weave.works/v1alpha2
 kind: CAPITemplate
```

Comment:
Are we not deprecating now with this change slowly CAPI templates? and merge them all under the umbrella of GitOpsTemplates?

Reply:
Yeah we should good shout